File: Telegram App/HomeViewController.m
//
//  HomeViewController.m
//  Telegram App
//
//  Created by Sandeep  Raghunandhan on 8/7/14.
//  Copyright (c) 2014 Sandeep  Raghunandhan. All rights reserved.
//

#import "HomeViewController.h"
#import "TelegramViewController.h"
#import "MorseReferenceTableViewController.h"
@interface HomeViewController()


@end
@implementation HomeViewController
-(instancetype)init{
    self = [super init];
    if (self)
    {
        UINavigationItem *navItem = self.navigationItem;
        navItem.title = @"Home";

    }
    return self;
}
- (IBAction)launchTranslator:(id)sender {
    TelegramViewController *tele = [[TelegramViewController alloc]init];
    [self.navigationController pushViewController:tele animated:YES];
}
- (IBAction)launchMorseReference:(id)sender {
    MorseReferenceTableViewController *instructions = [[MorseReferenceTableViewController alloc]init];
    [self.navigationController pushViewController:instructions animated:YES];
}

@end

File: Telegram App/HomeViewController.swift
//
//  HomeViewController2.swift
//  Telegram App
//
//  Created by Sandeep on 11/18/16.
//  Copyright © 2016 Sandeep. All rights reserved.
//

import Foundation
import UIKit

class HomeViewController: UIViewController {
    
    convenience init() {
        self.init(nibName:nil, bundle:nil)
        let navItem = self.navigationItem;
        navItem.title = "Home";
    }
    @IBAction func launchTranslator(_ sender: UIButton) {
        let tele = TelegramViewController();
        self.navigationController?.pushViewController(tele, animated: true);
        
    }
    @IBAction func launchReference(_ sender: UIButton) {
        let table = MorseReferenceTableViewController();
        self.navigationController?.pushViewController(table, animated: true);
    }

}
